ibis/backends/dask/execution/util.py

```diff
@@ -47,25 +50,126 @@ def make_selected_obj(gs: SeriesGroupBy):
         ]
 
 
-def maybe_wrap_scalar(result: Any, expr: ir.Expr) -> Any:
-    """
-    A partial implementation of `coerce_to_output` in the pandas backend.
+def coerce_to_output(
+    result: Any, expr: ir.Expr, index: Optional[pd.Index] = None
+) -> Union[dd.Series, dd.DataFrame]:
+    """Cast the result to either a Series of DataFrame, renaming as needed.
+
+    Reimplementation of `coerce_to_output` in the pandas backend, but
+    creates dask objects and adds special handling for dd.Scalars.
 
-    Currently only wraps scalars, but will change when udfs are added to the
-    dask backend.
+    Parameters
+    ----------
+    result: Any
+        The result to cast
+    expr: ibis.expr.types.Expr
+        The expression associated with the result
+    index: pd.Index
+        Optional. If passed, scalar results will be broadcasted according
+        to the index.
+
+    Returns
+    -------
+    result: A Series or DataFrame
```

Comment:
dd.Series & dd.DataFrame

Reply:
Hmmm....I can update the comment if you'd like, but I think the implicit assumption is inside the dask backend a `Series` or `DataFrame` is always referring to the dask versions.